sros2/sros2/_utilities.py

```diff
@@ -90,7 +95,7 @@ def build_key_and_cert(subject_name, *, ca=False, ca_key=None, issuer_name=''):
             # Using a day earlier here to prevent Connext (5.3.1) from complaining
             # when extracting it from the permissions file and thinking it's in the future
             # https://github.com/ros2/ci/pull/436#issuecomment-624874296
```

Comment:
I'm assuming this is not an issue anymore, shall we remove this then? 

```suggestion
```

Reply:
indeed :+1: the goal of this round of CI is to confirm if we can indeed remove those
(although the ticket was referencing issues on MacOS machines so we will not know for sure that the issue has been resolved on that platform)

Reply:
Thanks for the express review BTW :+1: